pages/candy-machines/[candy_machine_ID].tsx

```diff
@@ -37,6 +37,7 @@ const CandyMachine: NextPage = () => {
     const [isVerifyOpen, setIsVerifyOpen] = useState(false)
 
     const fetchNfts = async () => {
+        setNftsRecoilState([])
```

Comment:
We could use the recoil hook `useResetRecoilState` to set this to its default value ([]) and then replace the `nftsRecoilState` for a useRecoilValue with a selector that returns the length of the state, that is what we are only using this for in this file:

https://recoiljs.org/docs/api-reference/core/useRecoilValue
https://recoiljs.org/docs/api-reference/core/useResetRecoilState

Funcional-wise won't affect, but I think is a better approach to use the full stack armory from Recoil!

Reply:
Perfect!